src/api/registrationApi.ts

```diff
@@ -18,6 +18,16 @@ export const updateRegistration = async (registration: Registration) =>
     data: registration,
   });
 
+export const unpublishRegistration = async (
+  registration: Registration,
+  unpublishingRequest: UnpublishPublicationRequest
+) =>
+  await authenticatedApiRequest<Registration>({
```

Comment:
vi er i en laaaangvarig prosess hvor vi gradvis går bort fra tidligere måter å kommunisere med APIet på. I stedet bruker vi `react-query` som gir oss en enklere måte å håndtere loading, refetch, error, etc (les: vi slipper å gjøre det manuelt som før, som alltid vil gi fuckups her-og-der 😅)

```suggestion
  await authenticatedApiRequest2<Registration>({
```